.github/workflows/publish-command.yml

```diff
@@ -95,17 +95,24 @@ jobs:
         with:
           repository: ${{ github.event.inputs.repo }}
           ref: ${{ github.event.inputs.gitref }}
+          token: ${{ secrets.OCTAVIA_PAT }}
       - name: Install Java
         uses: actions/setup-java@v1
         with:
-          java-version: '17'
+          java-version: "17"
       - name: Install Pyenv and Tox
         # Beside PyEnv, this does not set any runtimes up because it uses an AMI image that has everything pre-installed. See https://github.com/airbytehq/airbyte/issues/4559/
         run: |
           python3 -m pip install --quiet virtualenv==16.7.9 --user
           python3 -m virtualenv venv
           source venv/bin/activate
           pip install --quiet tox==3.24.4
+      - name: Install yq
```

Comment:
nit:
Wondering what's the rationale behind using `yq`. As we already have python installed, could we reproduce the same find and replace logic in a python script (that might be a bit more readable) rather than in the bash commands in the steps below?

Reply:
~~because I'm a masochist~~ my thinking is that I didn't want to mess with the py venv by adding a yaml parser in order to manipulate the definitions, but it's a fair point and wouldn't be against it if it's totally preferable.